experiments/AMIP/modular/components/land/bucket_init.jl

```diff
@@ -181,14 +184,18 @@ function bucket_init(
     earth_param_set = create_lsm_parameters(FT)
 
     α_snow = FT(0.8) # snow albedo
-    if albedo_from_file # Read in albedo from data file (default)
-        albedo = BulkAlbedoMap{FT}(regrid_dirpath, α_snow = α_snow, comms = comms_ctx)
-    else # Use spatially-varying function for surface albedo
+    if albedo_type == "map_static" # Read in albedo from static data file (default)
+        albedo = BulkAlbedoStatic{FT}(regrid_dirpath, comms = comms_ctx)
+    elseif albedo_type == "map_temporal"
+        albedo = BulkAlbedoTemporal{FT}(regrid_dirpath, date_ref, t_start, space)
```

Comment:
I would add a bit more info on the files and data (is this linearly interpolated monthly data? What is the source of  the file?).